x-pack/plugins/security/server/authentication/index.test.ts

```diff
@@ -166,6 +166,34 @@ describe('setupAuthentication()', () => {
       expect(authenticate).toHaveBeenCalledWith(mockRequest);
     });
 
+    it('returns authentication response headers on success if any', async () => {
```

Comment:
__note:__ diff#1 with 7.3 version: we don't have auth_redirect in 7.x/master anymore, so the similar test is here now.